app/controllers/fuel/posts_controller.rb

```diff
@@ -3,13 +3,20 @@ class PostsController < FuelController
     include ActionView::Helpers::TextHelper
     layout Fuel.configuration.layout if Fuel.configuration.layout
     before_filter :define_title
+    if Fuel.configuration.header_pagination
+      paginated_action only: [:index]
```

Comment:
Suggest treating all calls to PostsController#index as pagination-aware calls. See my comments on `fuel_controller.rb`.